authoring-recipes/multiple-visitors.md

```diff
@@ -389,11 +389,11 @@ public class FinalizeLocalVariables extends Recipe {
 
                 // TODO: Return if any variables in the varDec are uninitialized
 
-                // TODO: Return if the varDec is in a for loop control
+                // TODO: Return if the varDec is in a for-loop control
 
                 // TODO: Return if the varDec is an instance or class variable
 
-                // TODO: If none of the above apply, find all use cases for the variables and determine whether or not they're reassigned. If none have been, add the final modifier. If one ore more has, do nothing.
+                // TODO: If none of the above apply, find all use cases for the variables and determine whether or not they're reassigned. If none have been, add the final modifier. If one or more has, do nothing.
```

Comment:
Interesting typo. Sadly [`ore`](https://www.dictionary.com/browse/ore) is a word, but `^ore$` could be added to https://github.com/openrewrite/rewrite-docs/blob/master/.github/actions/spelling/reject.txt if it's unlikely to be appropriate in this repository :)